RELEASE_NOTES.md

```diff
@@ -6,7 +6,11 @@ This page describes the noteworthy improvements provided by each release of Ecli
 
 ### Migration guide 2.x -> 3.x
 
-- ...
+#### Pack200
+
+Pack200 technology is obsolete and no longer supported after Java 13. Tycho removed all support for dealing with pack.gz files including pack200 specific plugins, various options in Mojos and support for resolving fetching in core. 
+
+Omit any pack200 specific plugins, options and in any other way dealing with ***.pack.gz** files.
```

Comment:
Do you think we can have a list of Mojos and Options that are removed?

Reply:
Sure, will do.